applications/NXsas.nxdl.xml

```diff
@@ -27,6 +27,18 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
+  <symbols>
+    <doc>
+	  The symbol(s) listed here will be used below to coordinate datasets
+	  with the same shape.
+	</doc>
+	<symbol name="nXPixel">
```

Comment:
apply consistent names and descriptions here (see NXreftof for similar comment)

Reply:
For these symbols that appear to be similar between definitions, the concern was that if we change them to the same symbol name across all definitions (from _xSize, nXPixel_ etc to **nX** for example) that there is the possibility that the community that uses the definition may become confused and wonder if the meaning of that symbol has changed. As I remember the discussion a decision either way was not reached, which I guess is the same as don't change it now?.